README.adoc

```diff
@@ -323,6 +323,29 @@ foo&. bar
 foo&.bar
 ----
 
+=== Safe navigation
```

Comment:
I'd argue regarding the section, Source code layout.
Maybe it would be a better fit for [Misc](https://rubystyle.guide/#misc)?

Reply:
Can you elaborate more regarding this comment?

Reply:
If I'm not mistaken, presently this new guideline resides in the "Source code layout" section. This section is mostly all about whitespace, formatting, those things that don't change code semantics.

The new guideline suggests code changes that may, depending on constraints, change its behaviour.

Basing on this observation, I propose moving this new guideline to a different section. I don't have a better section in mind, so I suggested "Misc".

Reply:
got it